### PR TITLE
US-7.5: Fix notification list display problem

### PR DESCRIPTION
**Pull request by AzmiTouil**

### CODE REVIEW

File: digital-workplace-webapps/src/main/webapp/vue-app/topBar/notification/components/NotificationApp.vue [hunk 1]
```diff
@@ -46,16 +46,17 @@
...
                   <v-layout>
-                    <v-flex xs8>
+                    <v-flex xs7>
                       <v-btn
                         style="border-radius: 1em;"
                         depressed
                         color="#E1E8EE"
-                        class="my-2 ml-2"><span class="font-weight-black">See all</span></v-btn></v-flex>
-                    <v-flex xs4>
+                        class="my-2 ml-2"><span>See all</span></v-btn></v-flex>
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/topBar/notification/components/NotificationApp.vue]: All labels should internationalized

Comment (AzmiTouil) [reply digital-workplace-webapps/src/main/webapp/vue-app/topBar/notification/components/NotificationApp.vue]: Done

File: digital-workplace-webapps/src/main/webapp/vue-app/topBar/notification/components/NotificationApp.vue [hunk 2]
```diff
@@ -46,16 +46,17 @@
...
                   <v-layout>
-                    <v-flex xs8>
+                    <v-flex xs7>
                       <v-btn
                         style="border-radius: 1em;"
                         depressed
                         color="#E1E8EE"
-                        class="my-2 ml-2"><span class="font-weight-black">See all</span></v-btn></v-flex>
-                    <v-flex xs4>
+                        class="my-2 ml-2"><span>{{ this.$t('homepage.seeAll') }}</span></v-btn></v-flex>
```
Comment (azayati) [inline digital-workplace-webapps/src/main/webapp/vue-app/topBar/notification/components/NotificationApp.vue]: Change it to homepage.topBar.notification.seeAll better

Comment (AzmiTouil) [reply digital-workplace-webapps/src/main/webapp/vue-app/topBar/notification/components/NotificationApp.vue]: Done

**Pull request merged**